addons/mail_plugin/models/ir_http.py

```diff
@@ -24,3 +24,10 @@ def _auth_method_outlook(cls):
 
         # take the identity of the API key user
         request.update_env(user=user_id)
+
+        # switch to the user context
+        new_context = {  # remove duplicated keys before calling update_context
+            **request.env.context,
+            **request.env.user.sudo().context_get(),
```

Comment:
sudo?! why?

Reply:
it was already in the previous pr in 15.0, I had issue while testing
shouldn't be an issue no ? (I can recheck if it's still needed in 16.0, but I need to push on the runbot because the domain is whitelisted on *.odoo.com)